bin/main/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Root logger option
```

Comment:
Did you manually create/intend to add this file? If so, why? If not, let's remove it -- it doesn't seem critical to addressing this particular failing unit test.